src/bin/pub.ts

```diff
@@ -101,65 +92,67 @@ export default function start(args: string[]) {
 		},
 	})
 
-	if (parsedArgs.help) {
+	if (parsedArgs['help']) {
 		return helpMe.toStdout('publish')
 	}
 
-	if (parsedArgs.key) {
-		parsedArgs.key = fs.readFileSync(parsedArgs.key)
+	if (parsedArgs['key']) {
+		parsedArgs['key'] = fs.readFileSync(parsedArgs['key'])
 	}
 
-	if (parsedArgs.cert) {
-		parsedArgs.cert = fs.readFileSync(parsedArgs.cert)
+	if (parsedArgs['cert']) {
+		parsedArgs['cert'] = fs.readFileSync(parsedArgs['cert'])
 	}
 
-	if (parsedArgs.ca) {
-		parsedArgs.ca = fs.readFileSync(parsedArgs.ca)
+	if (parsedArgs['ca']) {
+		parsedArgs['ca'] = fs.readFileSync(parsedArgs['ca'])
 	}
 
-	if (parsedArgs.key && parsedArgs.cert && !parsedArgs.protocol) {
-		parsedArgs.protocol = 'mqtts'
+	if (parsedArgs['key'] && parsedArgs['cert'] && !parsedArgs['protocol']) {
+		parsedArgs['protocol'] = 'mqtts'
 	}
 
-	if (parsedArgs.port) {
-		if (typeof parsedArgs.port !== 'number') {
+	if (parsedArgs['port']) {
+		if (typeof parsedArgs['port'] !== 'number') {
 			console.warn(
 				"# Port: number expected, '%s' was given.",
-				typeof parsedArgs.port,
+				typeof parsedArgs['port'],
 			)
 			return
 		}
 	}
 
 	if (parsedArgs['will-topic']) {
-		parsedArgs.will = {}
-		parsedArgs.will.topic = parsedArgs['will-topic']
-		parsedArgs.will.payload = parsedArgs['will-message']
-		parsedArgs.will.qos = parsedArgs['will-qos']
-		parsedArgs.will.retain = parsedArgs['will-retain']
+		parsedArgs['will'] = {}
+		parsedArgs['will'].topic = parsedArgs['will-topic']
+		parsedArgs['will'].payload = parsedArgs['will-message']
+		parsedArgs['will'].qos = parsedArgs['will-qos']
+		parsedArgs['will'].retain = parsedArgs['will-retain']
```

Comment:
All of these, and every access made that way can be easily resolved by using typeguards in a later PR.